sites/svelte.dev/src/routes/_components/Supporters/index.svelte

```diff
@@ -2,6 +2,11 @@
 	import { Section } from '@sveltejs/site-kit/components';
 	import contributors from './contributors.js';
 	import donors from './donors.js';
+
+	// @ts-ignore
+	import contributors_img from './contributors.jpg?w=1200&format=webp;jpg&as=picture';
```

Comment:
I don't see the jpg version used? So I think you could remove that

And then if you drop the `&as=picture` as well you can drop the `.sources.webp[0].src` below

Reply:
Done

Reply:
Done